apps/lfx-one/src/app/modules/dashboards/executive-director/components/social-media-drawer/social-media-drawer.component.html

```
@@ -0,0 +1,174 @@
+        <span class="text-sm text-gray-500">Total Followers</span>
+        <span class="text-2xl font-semibold text-gray-900">{{ formatNumber(data().totalFollowers) }}</span>
+      </div>
+      <div class="flex-1 flex flex-col gap-1 p-4 border border-gray-200 rounded-lg">
+        <span class="text-sm text-gray-500">Platforms Active</span>
+        <span class="text-2xl font-semibold text-gray-900">{{ data().totalPlatforms }}</span>
+      </div>
+      <div class="flex-1 flex flex-col gap-1 p-4 border border-gray-200 rounded-lg">
+        <span class="text-sm text-gray-500">Follower Growth</span>
+        <div class="flex items-center gap-2">
+          <span class="text-2xl font-semibold" [class]="data().trend === 'up' ? 'text-green-600' : 'text-red-600'">
+            {{ data().changePercentage > 0 ? '+' : '' }}{{ data().changePercentage }}%
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)